examples/gallery/embellishments/rose.py

```diff
@@ -0,0 +1,45 @@
+"""
+Rose diagram
+------------
+
+The :meth:`pygmt.Figure.rose` method can plot windrose diagrams or polar histograms.
+"""
+
+import pygmt
+
+data = pygmt.datasets.load_fractures_compilation()
+
+fig = pygmt.Figure()
+
+fig.rose(
+    # use the remote file fractures_06.txt which contains a compilation
+    # of fracture lengths and azimuth as hypothetically digitized from
+    # geological maps
+    data=data,
```

Comment:
Although `rose` can accept either `data`, or `length`/`azimuth` pairs, I feel that passing `length=data.length` and `azimuth=data.azimuth` is more readable than using `data=data, columns=[1, 0]`.

Reply:
Thanks, indeed much better.